openAI.sublime-settings

```diff
@@ -1,10 +1,7 @@
 {
-    // Custom URL for API (useful when using custom, OpenAI compattible API) in format host:port
-    "url": "api.openai.com",
-    
-    // Whether to use HTTPS for connection
-    "https": true,
-    
+    // Custom URL for API (useful when using custom, OpenAI compattible API)
```

Comment:
The very last bit, add the doc right here, to explain how it's work to a user.

To emphasise that `http(s)` prefix matters and when which to use.